src/mlpack/methods/kernel_svm/kernel_svm.hpp

```diff
@@ -216,7 +214,7 @@ class KernelSVM
   //! Locally saved KernelType values.
   KernelType kernel;
   //! Locally saved labels of classes to give prediction for non-linear kernel.
-  arma::rowvec trainLabels;
+  arma::Row<int> trainLabels;
```

Comment:
use `size_t` instead of int.
Could you please make this change everywhere else as well?